blocks/image-placeholder/index.js

```diff
@@ -25,13 +25,13 @@ import { rawHandler } from '../api';
  */
 export default function ImagePlaceholder( { className, icon, label, onSelectImage, multiple = false } ) {
```

Comment:
Do you think, this could be renamed "MediaPlaceholder" to make the experience even more similar? and provide a media type?

Reply:
As a next iteration, we could bring those two closer by doing proper refactoring. Actually, this was the first step I took, but then I backed off, because I didn't want to change too much in one PR.

Reply:
Also see my comment https://github.com/WordPress/gutenberg/pull/5431#issuecomment-370725660, you were too fast :)